addons/binding/org.openhab.binding.network/README.md

```diff
@@ -1,7 +1,7 @@
 # Network Binding
 
 This binding allows to check, whether a device is currently available on the network.
-This happens by either using [ping](https://en.wikipedia.org/wiki/Ping_%28networking_utility%29) or by a successful TCP connection on a port of your choosing.
+This happens by either using [ping](https://en.wikipedia.org/wiki/Ping_%28networking_utility%29), [arping](https://en.wikipedia.org/wiki/Arping) or by a successful TCP connection on a port of your choosing.
```

Comment:
How about this:

- **arping (optional):** Use arping instead of regular ping. Only works on Unix and arping needs to be installed. By setting this option to `true`, the **port option is ignored** and it is **required to specify a network interface**. Arping works on a lower level then a regular ping and might get a reply when a regular ping does not. This might be the case when a device with a recent version of Android goes to sleep.

I don't know much more about arping, besides that it fixed my ping issues with my Galaxy S7.